### ISSUE TITLE
Glitchy source text - correct, or re-extract?

### ISSUE THREAD

**Issue by tlongers** [official-inquiries/uk-parliament-20160706-iraq-inquiry#4]
PDF to text creates jumbled text sometimes e.g.

Original source:
![the-report-of-the-iraq-inquiry_introduction_pdf__page_2_of_19_](https://cloud.githubusercontent.com/assets/2115933/16689509/3ad22706-451b-11e6-93eb-63fd102cb400.png)

Markdown formatted preview:

![introduction_md__80__](https://cloud.githubusercontent.com/assets/2115933/16689529/4e862dec-451b-11e6-9e57-af2d41fbe6c8.png)

Correct the issue when formatting in MD, or attempt re-extraction and hope for a better outcome?

**Comment by igorbrigadir** [comment]
@cr3ative has a good cleaned up version https://github.com/cr3ative/chilcot-html in case that's useful to compare (no sure how it was extracted though)

**Comment by cr3ative** [comment]
@igorbrigadir This is the output of all the PDFs concatenated then exported to HTML using Acrobat Pro CC. The index page has an extra search box pointed at GitHub, but that's about it.

**Comment by igorbrigadir** [comment]
Thanks! That might be a good pipeline - might be easier to clean up the HTML result rather than the extracted text from PDF - selecting elements (footers, headers, bullets) by style.

**Comment by rufuspollock** [comment]
@igorbrigadir good point. Can you let us know how you get on. I've taken a quick look at the @cr3ative version but not done an in depth comparison vs simple text extraction.
